### PR TITLE
ENT-9167: Added ci docker scripts for manual and github actions use

### PR DESCRIPTION
**Pull request by craigcomstock**
https://github.com/cfengine/mission-portal/pull/1996

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @Lex-2008 can review this?

**Comment by craigcomstock** [comment]
@Lex-2008 I switched to this PR in-repo so that the workflow can run. Should be nearly the same as what you already reviewed. :)

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/9649/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/9649/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-9649/

File: ci/docker.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# run the build in a docker container
+set -ex
+
+git status # for debugging github action checkout lagging maybe?
+git log -4
+echo "written at 13:11:07 by craig"
```
Comment (Lex-2008) [inline ci/docker.sh:7]: cozy :) Especially when you say time (13:11), but omit date ;)

**Review by Lex-2008 — DISMISSED** [review]

File: ci/docker.sh (hunk 1 above)
Comment (craigcomstock) [reply ci/docker.sh:7]: oh dang. madness. :( will correct.

Comment (craigcomstock) [reply ci/docker.sh:7]: thank you for reading carefully :)

**Comment by craigcomstock** [comment]
CI failure is a timing flake. Will correct one debug chunk and then we can merge this and https://github.com/cfengine/mission-portal/pull/1996

**Review by Lex-2008 — APPROVED** [review]

**Comment by craigcomstock** [comment]
cherry picks to
3.18.x: https://github.com/cfengine/buildscripts/pull/1244
3.21.x: https://github.com/cfengine/buildscripts/pull/1245

They might need some adjustments/fixes for github cache naming due to branch instead of master.

**Pull request merged**